doc/development/guide/installation.rst

```diff
@@ -66,6 +66,84 @@ importing PennyLane in Python.
     requires ``pip install -e .`` to be re-run in the plugin repository
     for the changes to take effect.
 
+Poetry
```

Comment:
Please could we update the installation section above to recommend use of poetry?

Reply:
I'm not a fan of us explicitly endorsing Poetry in this way. I think this needs to be reworded as "supports poetry" rather than explicitly want people to use this. This isn't something we'll be adding to Lightning or Catalyst, so making it an optional utility instead is what I'd prefer. Can we make this update?